torch/_dynamo/variables/builtin.py

```diff
@@ -931,6 +933,13 @@ def call_isinstance(self, tx, arg, isinstance_type):
             val = arg_type is isinstance_type
         return variables.ConstantVariable(val)
 
+    def call_issubclass(self, tx, left_ty, right_ty):
+        """Checks if first arg is subclass of right arg"""
+        left_ty = left_ty.as_python_constant()
+        right_ty = right_ty.as_python_constant()
+
+        return variables.ConstantVariable(issubclass(left_ty, right_ty))
```

Comment:
Pretty sure you need an `add_options` here??

Reply:
It's strange to me that this has to be done so manually, I would hope that in general, it would be a one-liner to add support for any non-side-effecting builtin that is operating on constants entirely. Having to manually define a `call_` per definitely seems suboptimal.

Reply:
I added the options, I think you're right, but there are different patterns to arguments for a lot of builtins, and I'm following the existing pattern

Reply:
But you didn't `add_options`

Reply:
Added now fr